darts/models/theta.py

```diff
@@ -95,7 +95,7 @@ def fit(self, series: TimeSeries, component_index: Optional[int] = None):
             new_ts = remove_from_series(ts, self.seasonality, model=self.season_mode)
 
         # SES part of the decomposition.
-        self.model = hw.SimpleExpSmoothing(new_ts.values()).fit()
+        self.model = hw.SimpleExpSmoothing(new_ts.values()).fit(initial_level=0.2)
```

Comment:
Do we want to change this in this PR?

Reply:
Sorry, it was something that I forgot to clean. 
Same for the comments after that